icu_benchmarks/models/ml_models/catboost.py

```
@@ -0,0 +1,19 @@
+from icu_benchmarks.contants import RunMode
+from icu_benchmarks.models.wrappers import MLWrapper
+
+
+@gin.configurable
+class CBClassifier(MLWrapper):
+    _supported_run_modes = [RunMode.classification]
+
+    def __init__(self, *args, **kwargs):
+        # self.model = self.set_model_args(cb.CatBoostClassifier, task_type="GPU"
+        # if not kwargs['cpu'] else "CPU", *args, **kwargs)
+        self.model = self.set_model_args(cb.CatBoostClassifier, task_type="CPU", *args, **kwargs)
```

Ruff

Star-arg unpacking after a keyword argument is strongly discouraged

(B026)